crates/forge_infra/src/env.rs

```diff
@@ -109,8 +109,9 @@ impl ForgeEnvironmentService {
     }
 
     fn get(&self) -> Environment {
-        dotenv::dotenv().ok();
         let cwd = std::env::current_dir().unwrap_or(PathBuf::from("."));
+        Self::dot_env(&cwd);
```

Comment:
Should be called only once. Can use a `RWLock<bool>` where the bool represents if the env has been loaded or not.